doc/source/examining/loading_data.rst

```diff
@@ -164,8 +164,9 @@ Appropriate errors are thrown for other combinations.
 * particle data: currently not supported (but might come later)
 * staggered grids (AMRVAC 2.2 and later): yt logs a warning if you load
   staggered datasets, but the flag is currently ignored.
-* "stretched grids" as defined in AMRVAC have no correspondence in yt,
-  hence will never be supported.
+* "stretched grids" are being implemented in yt, but are not yet
+  fully-supported.  (Previous versions of this file suggested they would
+  "never" be supported, which we hope to prove incorrect.)
```

Comment:
oopsy. Nice catch :)

Reply:
Maybe worth specifying _why_ they are not supported yet for AMRVAC (missing compatibility with multi grids and AMR)

Reply:
still relevant (Sorry GitHub is such a pain with long discussions)